### ISSUE TITLE
[Conduct Test] Patient search results do not go away when tabbing away

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6363]
## Area or Component

[where is this happening? the navigation bar, the patient experience, static site, etc?]: #
Conduct Test

## Quick Summary

[what's wrong here?]: #
Patient search results appear over other tests without going away. The only way to remove this popup is to clear the search textbox.

## Accessibility Violation

[why is this inaccessible? include deque link if available.]: #

## Action Requested

[how can we fix this?]: #
convert the results to a modal? I'm not actually quite sure.

## Screenshots (optional)

https://github.com/CDCgov/prime-simplereport/assets/10108172/77ffa924-3eeb-4980-a2bb-1d7f3082d772

## Source Code (optional)

[which file or line is the issue attached to?]: #
